tasker_tutorail/c02_task_folder/Taskfile.yml

```diff
@@ -0,0 +1,31 @@
+# to create a Taskfile.yaml in a dircetory, run task -i or task --init
+# https://taskfile.dev
+
+version: '3'
+
+vars:
+  GREETING: 'By default the task runs in the same dircetory where the Taskfile is located, but you can write a custom dircetory for any particular task to run in'
```

Comment:
please use 79 chars per line (you can configure your editor to show you the ruler)
so I won't need to scroll the page horizontally or using word wrap
in this case I propose to use YAML multi line syntax:
```
GREETING: |-
  By default the task runs in the same dircetory where the Taskfile is located
  but you can write a custom dircetory for any particular task to run in
```
its much more readable

Reply:
fixed by moving all the long lines to .md text files

Reply:
it was great in the GReetings